recipes/angelscript/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conans import CMake, ConanFile, tools
+
+import os
+import shutil
+
+
+class AngelScriptConan(ConanFile):
+    name = "angelscript"
+    license = "Zlib"
+    homepage = "http://www.angelcode.com/angelscript"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "An extremely flexible cross-platform scripting library designed to allow applications to extend their functionality through external scripts."
+    topics = "angelcode", "embedded", "scripting", "language", "compiler", "interpreter"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [False, True],
+        "fPIC": [False, True],
+        "no_exceptions": [False, True],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "no_exceptions": False,
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.configure(
+            args=[
+                "-DCMAKE_CXX_STANDARD=11",
+                "-DAS_NO_EXCEPTIONS={}".format(self.options.no_exceptions.value),
+            ],
+            source_folder=os.path.join(self._source_subfolder, "angelscript", "projects", "cmake"),
```

Comment:
I seems you are missing our `cmake_wrapper` which is required to set all the CMake setting to the Conan build settings 

The may not be generating static runtime binaries for example

Reply:
Yes, please add CMake wrapper